app/code/Magento/Checkout/Test/Unit/Block/Cart/ShippingTest.php

```diff
@@ -3,122 +3,215 @@
      */
-    protected $model;
+    protected $block;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var Context|MockObject
      */
-    protected $context;
+    protected $contextMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var customerSession|MockObject
      */
-    protected $customerSession;
+    protected $customerSessionMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var checkoutSession|MockObject
      */
-    protected $checkoutSession;
+    protected $checkoutSessionMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var CompositeConfigProvider|MockObject
      */
-    protected $configProvider;
+    protected $configProviderMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var LayoutProcessorInterface|MockObject
      */
-    protected $layoutProcessor;
+    protected $layoutProcessorMock;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var StoreManagerInterface|MockObject
      */
-    protected $storeManager;
+    protected $storeManagerMock;
 
     /**
      * @var array
      */
     protected $layout;
 
     /**
-     * @var \PHPUnit_Framework_MockObject_MockObject
+     * @var Json|MockObject
      */
-    private $serializer;
+    private $serializerMock;
 
-    protected function setUp()
+    /**
+     * @var JsonHexTag|MockObject
+     */
+    private $jsonHexTagSerializerMock;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp(): void
     {
-        $this->context = $this->createMock(\Magento\Framework\View\Element\Template\Context::class);
-        $this->customerSession = $this->createMock(\Magento\Customer\Model\Session::class);
-        $this->checkoutSession = $this->createMock(\Magento\Checkout\Model\Session::class);
-        $this->configProvider = $this->createMock(\Magento\Checkout\Model\CompositeConfigProvider::class);
-        $this->layoutProcessor = $this->createMock(\Magento\Checkout\Block\Checkout\LayoutProcessorInterface::class);
+        $this->contextMock = $this->createMock(Context::class);
+        $this->customerSessionMock = $this->createMock(customerSession::class);
+        $this->checkoutSessionMock = $this->createMock(checkoutSession::class);
+        $this->configProviderMock = $this->createMock(CompositeConfigProvider::class);
+        $this->layoutProcessorMock = $this->createMock(LayoutProcessorInterface::class);
+        $this->serializerMock = $this->createMock(JsonHexTag::class);
+        $this->jsonHexTagSerializerMock = $this->createMock(JsonHexTag::class);
+        $this->storeManagerMock = $this->createMock(StoreManagerInterface::class);
         $this->layout = [
             'components' => [
-                'firstComponent' => ['param' => 'value'],
-                'secondComponent' => ['param' => 'value'],
+                'firstComponent' => ['param' => 'value']
             ]
```

Comment:
Extract that array into `private const STUB_PREINITIALIZED_COMPONENTS`

Reply:
sure, I will update

Reply:
@lbajsarowicz I have updated the changes. please review now.